libraries/botframework-luis/.gitignore

```diff
@@ -1,5 +1,6 @@
 /**/node_modules
 /**/.vscode
+/**/lib
```

Comment:
The `lib` folder for botrframework-luis still [exists](https://github.com/Microsoft/botbuilder-js/tree/stevenic/remove-libs/libraries/botframework-luis/lib), should we be adding it to the `.gitignore`?